src/Lexique/CompileJson.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Tintin\Lexique;
+
+trait CompileJson
+{
+    /**
+     * Compile the %json statement
+     *
+     * @param string $expression
+     * @return string
+     */
+    protected function compileJson(string $expression): string
+    {
+        $output = preg_replace_callback(
+            '/^%json\((.*)\)$/',
```

Comment:
Sometimes the developers write directives like `%json ()`. You must consider the white spacing between `%json ` and `()`

Reply:
Thanks, I forgot that detail.